### PR TITLE
[CKW-47] Badge Contact List Component

### PR DESCRIPTION
**Pull request by PanpanPianchoopat**
[https://cpe-kmutt-web.atlassian.net/browse/CKW-47?atlOrigin=eyJpIjoiZTI1NWM3MmQ3Mzg0NGRkOWI2MjI5YTZlNjA1ZjhlNGYiLCJwIjoiaiJ9](url)

### CODE REVIEW

File: frontend/src/common/components/BadgeContact/index.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { HiOutlineArrowRight } from 'react-icons/hi';
+
+import Button from '../Button';
+import Container from '../Container';
```
Comment (nitpum) [inline frontend/src/common/components/BadgeContact/index.tsx]: please use absolute path like `common/components/Button`

File: frontend/src/common/components/BadgeContact/styled.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+export const BadgeList = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  font-family: 'Kanit';
+  background: #f5f5f5;
+  margin: 65px 0;
+`;
+export const Badge = styled.div`
```
Comment (nitpum) [inline frontend/src/common/components/BadgeContact/styled.ts]: insert space between export
```suggestion
`;

export const Badge = styled.div`
```

File: frontend/src/common/components/BadgeContact/styled.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+export const BadgeName = styled.p`
+  font-size: 20px;
+  font-weight: 700;
+  margin: 0;
+  @media (max-width: ${BREAKPOINTS.IPAD_PORTRAIT}) {
+    font-size: 16px;
+  }
+`;
+export const Description = styled.p`
```
Comment (nitpum) [inline frontend/src/common/components/BadgeContact/styled.ts]: here too

File: frontend/src/common/components/BadgeContact/index.tsx [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import {
+  BadgeListContainer,
+  BadgeList,
+  Badge,
+  BadgeIcon,
+  BadgeName,
+  Description,
+} from './styled';
+
+const BadgeContact: React.FC = () => {
```
Comment (nitpum) [inline frontend/src/common/components/BadgeContact/index.tsx]: should rename to something like `BadgeContacts` <- wthi `s`   or `BadgeContactSection`

**Review by nitpum — APPROVED** [review]
LGTM

File: frontend/src/common/components/BadgeContact/index.tsx (hunk 4 above)
Comment (nitpum) [inline frontend/src/common/components/BadgeContact/index.tsx]: I think this component should move to `src/modules/contact/components/`

**Comment by yeeezsh** [comment]
please also add this component to storybook

**Review by yeeezsh — APPROVED** [review]

**Pull request merged**